cri/stream/portforward/httpstream.go

```diff
@@ -1,6 +1,7 @@
 package portforward
 
 import (
+	con "context"
```

Comment:
I think we should use the `context` here to make the word readable, because the `con` might be the `connection`.